tests/ctst/common/common.ts

```diff
@@ -66,3 +78,27 @@ Given('an object {string} that {string}',
             ));
         }
     });
+
+Then('kafka consumed messages should not take too much place on disk',
+    async function (this: Zenko) {
+        const notToCheckTopics = ['oplog', 'dead-letter'];
+        const kafkaAdmin = new Kafka({ brokers: [this.parameters.KafkaHosts] }).admin();
+        const topics: string[] = (await kafkaAdmin.listTopics())
+            .filter(t => (t.includes(this.parameters.InstanceID) &&
+            !notToCheckTopics.some(e => t.includes(e))));
+        const previousOffsets = await getTopicsOffsets(topics, kafkaAdmin);
+
+        await Utils.sleep(35000);
```

Comment:
what is this delay for / related to?

* this definitely needs a comment to explain
* ideally, we don't want a hard-coded value but to derive/sync this value with its 'root cause' : e.g. either automatically read KFKC period from the cluster, or allow the delay to be configured to match